os/Kconfig.debug

```diff
@@ -529,6 +529,45 @@ config DEBUG_TTRACE
 	---help---
 		Enable T-trace debug SYSLOG output (disabled by default)
 
+comment "System Debug Options"
+
+config DEBUG_SYSTEM
+	bool "System debug"
+	default n
+	---help---
```

Comment:
You should make a dependancy with DEBUG. Because you use "dbg" in kernel/debug.

Reply:
DEBUG_SYSTEM is already inside DEBUG , however i added dependency with DEBUG_VERBOSE
if DEBUG 
config DEBUG_SYSTEM
endif

Reply:
Ok. Done. I have selected DEBUG_ERROR